adodb-memcache.lib.inc.php

```diff
@@ -401,6 +401,9 @@ public function flushAll($debug=false)
 			return false;
 
 		$del = $this->memcacheLibrary->flush();
+			if ($debug)
+				if (!$del) ADOConnection::outp("flushcache: $filename entry doesn't exist on memcache server!<br>\n");
+				else ADOConnection::outp("flushcache: $filename entry flushed from memcache server!<br>\n");
```

Comment:
Why add this ? It's the same logic as the test just below, and the $filename variable referenced in the message does not exist.

Also: incorrect indentation level and code does not follow guidelines